packages/web-components/src/components/cbp-form-input/cbp-form-field.scss

```diff
@@ -0,0 +1,235 @@
+      --cbp-form-field-color-border-focus: var(--cbp-color-interactive-secondary-dark);
+      --cbp-form-field-color-placeholder: transparent;
+      font-style: italic;
+      cursor: not-allowed;
+    }
+
+    &:disabled {
+      --cbp-form-field-color: var(--cbp-color-interactive-disabled-dark);
+      --cbp-form-field-color-bg: var(--cbp-color-interactive-disabled-light);
+      --cbp-form-field-color-border: var(--cbp-color-interactive-disabled-dark);
+      --cbp-form-field-color-border-hover: var(--cbp-color-interactive-disabled-dark);
+      //--cbp-form-field-color-border-focus: var(--cbp-color-interactive-disabled-dark);
+      --cbp-form-field-color-dark: var(--cbp-color-interactive-disabled-light);
+      --cbp-form-field-color-bg-dark: var(--cbp-color-interactive-disabled-dark);
+      --cbp-form-field-color-border-dark: var(--cbp-color-interactive-disabled-light);
+      --cbp-form-field-color-border-hover-dark: var(--cbp-color-interactive-disabled-light);
+      //--cbp-form-field-color-border-focus: var(--cbp-color-interactive-secondary-dark);
+      --cbp-form-field-color-placeholder: transparent;
+      font-style: italic;
+      cursor: not-allowed;
+    }
+  }
+  
+  input::placeholder {
+    color: var(--cbp-form-field-color-placeholder);
+    font-style: italic;
+  }
+
+  textarea { 
+    min-height: 6.75rem;
+    max-height: 80vh;
+    padding-block: var(--cbp-space-2x);
+    resize: vertical;
+  }
+
+  select {
+    //-webkit-appearance: none;
+    //-moz-appearance: none;
+    appearance: none;
+    text-overflow: ellipsis;
+    padding-block: calc(var(--cbp-space-2x) - 1px) ; // Fixes Firefox vertical alignment
+
+    option {
+      // only allows color and bgcolor to be changed in most browsers - these seem to be inherited in Chrome and Firefox already (not safari)
+      color: var(--cbp-form-field-color);
+      background-color: var(--cbp-form-field-color-bg);
+    }
+
+    // Safari is lacking in contrast, so these need to be manually set
+    optgroup {
+      color: var(--cbp-form-field-color);
+      font-weight: var(--cbp-font-weight-bold);
+    }
+  }
+
+  select:not([multiple]):not([size]) {
+    // Simple chevron replacement, no background
+    //background: right var(--cbp-space-2x) top var(--cbp-space-3x) no-repeat var(--cbp-form-field-color-bg) var(--cbp-form-field-select-chevron);
+    //padding-inline-end: var(--cbp-space-5x);
+
+    // Chevron on a colored background (via linear gradient)
+    background: right calc(1.125rem - 8px) top calc(1.125rem - 8px) no-repeat var(--cbp-form-field-select-chevron), 
+                right 0px top 0px repeat-y linear-gradient( 90deg, var(--cbp-form-field-color-bg) calc(100% - var(--cbp-space-9x)), var(--cbp-form-field-color-border) var(--cbp-space-9x) );
+    padding-inline-end: var(--cbp-space-9x);
+    
+  }
+
+  &[error] {
+    //--cbp-form-field-color: var(--cbp-color-text-darkest);
```

Comment:
Do these comments need to be removed or are they documentation?

Reply:
I'll keep them for now - just copied them all to keep track of the full set and which ones need to be overridden in my cases - could change with additions later.